### ISSUE TITLE
Sync error

### ISSUE THREAD

**Issue by tomdoo** [jotta/jotta-cli-issues#145]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
0.10.43842 unstable

**Description of problem:**
I'm using jotta-cli on a docker container with a raspberry pi 2.
When I enable sync I get this message : 
```
ERROR  Panic detected: runtime error: invalid memory address or nil pointer dereference
```

**jotta-cli status (`jotta-cli status`):**
```
------------------------------------------------------------------------------
 Account   :  on Jottacloud
 Usage     : 0bytes / 0bytes 
 Device    : jotta-cli-docker
------------------------------------------------------------------------------
 Sync is not enabled 
------------------------------------------------------------------------------
 No folders added to backup yet
------------------------------------------------------------------------------
OK
```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
```
pid:1 2021/05/08 18:39:39 jottad.sync.error: Panic detected: runtime error: invalid memory address or nil pointer dereference
git.jotta.us/jotta/desktop/client/sync/app.(*App).loop.func1
	/builds/jotta/desktop/client/sync/app/app.go:406
runtime.gopanic
	/usr/local/go/src/runtime/panic.go:679
runtime.panicmem
	/usr/local/go/src/runtime/panic.go:199
runtime.sigpanic
	/usr/local/go/src/runtime/signal_unix.go:394
runtime/internal/atomic.goXadd64
	/usr/local/go/src/runtime/internal/atomic/atomic_arm.go:103
git.jotta.us/jotta/desktop/client/sync.(*ActionGroup).perform.func1
	/builds/jotta/desktop/client/sync/actiongroup.go:106
git.jotta.us/jotta/desktop/client/sync.(*ActionGroup).perform
	/builds/jotta/desktop/client/sync/actiongroup.go:111
git.jotta.us/jotta/desktop/client/sync.(*ActionGroup).Download
	/builds/jotta/desktop/client/sync/actiongroup.go:78
git.jotta.us/jotta/desktop/client/sync.(*Sync).PerformActions
	/builds/jotta/desktop/client/sync/sync.go:611
git.jotta.us/jotta/desktop/client/sync.(*Sync).sync
	/builds/jotta/desktop/client/sync/sync.go:526
git.jotta.us/jotta/desktop/client/sync.(*Sync).Start
	/builds/jotta/desktop/client/sync/sync.go:198
git.jotta.us/jotta/desktop/client/sync/app.(*App).loop
	/builds/jotta/desktop/client/sync/app/app.go:426
runtime.goexit
	/usr/local/go/src/runtime/asm_arm.s:868
```

**Comment by Kimbsen** [comment]
Thanks for the report. We will release a fix for this asap.

**Comment by Kimbsen** [comment]
We released 0.10.43926 which should resolve this crash.
